src/img/cap-encoding-xlen32.edn

```diff
@@ -6,11 +6,12 @@
 (def left-margin 100)
 (def right-margin 100)
 (def boxes-per-row 32)
-(draw-column-headers {:height 50 :font-size 26 :labels (reverse ["0" "1" "2" "" "" "" "" "" "" "9" "10" "11" "12" "" "" "" "" "17" "18" "19" "20" "21" "" "" "24" "25" "" "" "" "29" "30" "31"])})
+(draw-column-headers {:height 50 :font-size 26 :labels (reverse ["0" "1" "2" "" "" "" "" "" "" "9" "10" "11" "12" "" "" "" "" "17" "18" "19" "20" "21" "" "23" "24" "25" "" "" "" "29" "30" "31"])})
 
 (draw-box "SDP"         {:span 2})
 (draw-box "AP, M"       {:span 5})
-(draw-box "Reserved"    {:span 4})
+(draw-box "CL"          {:span 1})
+(draw-box "Reserved"    {:span 3})
```

Comment:
Could you reduce the font size for this image? Some of the labels are overflowing the fields.